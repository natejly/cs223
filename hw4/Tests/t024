#!/bin/bash
# Uses supplied hash function

trap "/usr/bin/killall -q -u $USER ./GmapUnit 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./GmapUnit ]; then
  echo './GmapUnit is missing or not executable'
  echo './GmapUnit is missing or not executable' 1>&2
  exit 1
fi

$BIN/run -stdout=stdout.out -stderr=/dev/null ./GmapUnit 2 </dev/null
cat stdout.out | head -n 1
