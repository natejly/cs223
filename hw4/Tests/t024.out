And now a message from your friendly local hash function: hello!
